packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1754,6 +1761,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
                     undefined,
                     version.id,
                     this.summaryTracker.referenceSequenceNumber,
+                    new ChildLogger(summaryLogger, undefined, { safeSummary: true }),
```

Comment:
Future: I'd like to see the RunningSummarizer's logger be able to look up `safe` like it does `summarizerRunTag` so this won't be necessary but will be included automatically.